pkg/odo/cli/dev/dev.go

```diff
@@ -7,11 +7,15 @@ import (
 	"os"
 	"path/filepath"
 
+	_component "github.com/redhat-developer/odo/pkg/component"
```

Comment:
the same as in https://github.com/redhat-developer/odo/pull/5576/files#r835158058

Reply:
updated